<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqLiveDataPanel</class>
 <widget class="QWidget" name="pqLiveDataPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="useDiskRadio">
     <property name="text">
      <string>Use disk</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="useNetworkRadio">
     <property name="text">
      <string>Use network</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="directoryLayout">
     <item>
      <widget class="QLineEdit" name="directoryEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>/source/paraview/build/live-output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseDirectoryButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="portLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="portSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="snapToLatestCheckBox">
     <property name="text">
      <string>Snap to latest time step</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="timeStepList"/>
   </item>
   <item>
    <widget class="QPushButton" name="createButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <connections/>
</ui>
